### PR TITLE
Basic game - human guessing against computer

### PR DESCRIPTION
**Pull request by breadoliveoilsalt**
Improved class design and tests. Computer's ships are hard coded in the first five rows, left justification.

### CODE REVIEW

File: lib/ai.rb [hunk 1]
```diff
@@ -0,0 +1,68 @@
+require 'require_all'
+require_all 'lib'
+
+class AI
+
+  def assign_coordinates_to_ships(fleet_placement_board)
```
Comment (marosluuce) [inline lib/ai.rb]: I find this hard to follow because it mixes board and ship knowledge. It would be clearer to add a whole ship to the board rather than reaching into the board to manipulate individual coordinates.

Comment (breadoliveoilsalt) [reply lib/ai.rb]: Roger that, thanks.  Agree it doesn't look great.  I'll see if I can clean up if I end up redoing the data and AI.
